apps/web/pages/signup.tsx

```diff
@@ -66,16 +68,6 @@ const FEATURES = [
   },
 ];
 
-const getOrgUsernameFromEmail = (email: string, autoAcceptEmailDomain: string) => {
-  const [emailUser, emailDomain = ""] = email.split("@");
-  const username =
-    emailDomain === autoAcceptEmailDomain
-      ? slugify(emailUser)
-      : slugify(`${emailUser}-${emailDomain.split(".")[0]}`);
-
-  return username;
-};
-
```

Comment:
Moved it to a separate module for reusability across client side and server side.